pkg/config/config.go

```diff
@@ -8,24 +8,33 @@ package config
 
 import (
 	"crypto/rand"
+	"fmt"
 
 	"github.com/golang/protobuf/proto"
 	"github.com/hyperledger/fabric-protos-go/common"
 )
 
 // SignConfigUpdate signs the given configuration update with a
-// specific signer.
-func SignConfigUpdate(configUpdate *common.ConfigUpdate, signer *Signer) (*common.ConfigSignature, error) {
-	signatureHeader, err := signer.CreateSignatureHeader()
+// specific signingIdentity.
+func SignConfigUpdate(configUpdate *common.ConfigUpdate, signingIdentity *SigningIdentity) (*common.ConfigSignature, error) {
+	signatureHeader, err := signingIdentity.CreateSignatureHeader()
 	if err != nil {
 		return nil, err
 	}
 
+	sh, err := proto.Marshal(signatureHeader)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal signature header: %v", err)
```

Comment:
Nit. Need to avoid `fmt.Errorf()` as per [error handling guideline](https://hyperledger-fabric.readthedocs.io/en/release-2.0/error-handling.html). Use `github.com/pkg/errors` package instead.

Reply:
We're treating this package as a library and are avoiding as many dependencies as possible as well as fabric-specific error/logging preferences.

Reply:
As Will stated, we are avoiding dependencies and projecting our coding standards into consuming environments.

Reply:
Got it. Thanks, @wlahti.